t/t0013-config-file.t

```diff
@@ -212,4 +212,75 @@ test_expect_success '[bootstrap] curve_cert file must be mode o-r' '
 	test_must_fail flux broker ${ARGS} -c conf14 /bin/true
 '
 
+#
+# [tbon] tests
+#
+
+test_expect_success 'tbon.keepalive is enabled and using sysctl defaults' '
+	cat <<-EOT >attrs_keepalive.exp &&
+	tbon.keepalive_count                    -1
+	tbon.keepalive_enable                   1
+	tbon.keepalive_idle                     -1
+	tbon.keepalive_interval                 -1
+	EOT
+	flux broker ${ARGS} flux lsattr -v >attrs.out &&
+	grep "^tbon.keepalive_" attrs.out | sort >attrs_keepalive.out &&
+	test_cmp attrs_keepalive.exp attrs_keepalive.out
+'
+
+test_expect_success 'tbon.keepalive values can be set via config' '
+
+	cat <<-EOT2 >attrs_keepalive2.exp &&
+	tbon.keepalive_count                    42
+	tbon.keepalive_enable                   0
+	tbon.keepalive_idle                     43
+	tbon.keepalive_interval                 44
+	EOT2
+	mkdir conf15 &&
+	cat <<-EOT >conf15/tbon.toml &&
+	[tbon]
+	keepalive_enable = 0
+	keepalive_count = 42
+	keepalive_idle = 43
+	keepalive_interval = 44
+	EOT
+	flux broker ${ARGS} -c conf15 flux lsattr -v >attrs2.out &&
+	grep "^tbon.keepalive_" attrs2.out | sort >attrs_keepalive2.out &&
+	test_cmp attrs_keepalive2.exp attrs_keepalive2.out
+'
+
+test_expect_success 'tbon.keepalive_enable command line overrides config' '
+	cat <<-EOT >attrs_keepalive3.exp &&
+	tbon.keepalive_count                    5
+	tbon.keepalive_enable                   1
+	tbon.keepalive_idle                     30
+	tbon.keepalive_interval                 1
+	EOT
+	flux broker ${ARGS} -c conf15 \
+		-Stbon.keepalive_enable=1 \
+		-Stbon.keepalive_idle=30 \
+		-Stbon.keepalive_interval=1 \
+		-Stbon.keepalive_count=5 \
+		flux lsattr -v >attrs3.out &&
+	grep "^tbon.keepalive_" attrs3.out | sort >attrs_keepalive3.out &&
+	test_cmp attrs_keepalive3.exp attrs_keepalive3.out
+'
+
+test_expect_success 'tbon.keepalive_enable attr value check works' '
```

Comment:
Not sure if adding checks for values of other keepalive attributes/config would be too onerous here.

Reply:
Yeah, I was being a bit lazy there :-)   OK, will add.